tools/bootstrap-librdkafka.sh

```diff
@@ -35,11 +35,11 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 ./configure --clean
 make clean
 
-if [[ "$OSTYPE" == "linux"* ]]; then
-  MKL_OPTS="--disable-gssapi"  
+if [[ $OSTYPE == "linux"* ]]; then
```

Comment:
micro nit: you dont need to quote on the right hand side either as long as it is just a single word. (but dont change it, just pointing it out)